docs/en/05_Contributing/03_Documentation.md

```diff
@@ -3,45 +3,35 @@ summary: Writing guide for contributing to SilverStripe developer and CMS user h
-repository and press the "edit" button. **You will need a free GitHub account to do this**. 
+The easiest way of editing any documentation is by clicking the "Edit this page" link at the bottom of the 
+page you want to edit. Alternatively, locate the appropriate .md file in the 
+[github.com/silverstripe/silverstripe-framework](https://github.com/silverstripe/silverstripe-framework/tree/master/docs/) repository and press the "edit" button. **You will need a free GitHub account to do this**. 
 
 
- * After you have made your change, describe it in the "commit summary" and "extended description" fields below, and 
- press "Commit Changes".
- * After that you will see form to submit a Pull Request.  You should be able to adjust the version your document ion change is for and then submit the form. Your changes 
- will be sent to the core committers for approval.
+ * After editing the documentation, describe your changes in the "commit summary" and "extended description" fields below then press "Commit Changes".
+ * After that you will see a form to submit a Pull Request. You should be able to adjust the version your document ion changes are for and then submit the form. Your changes will be sent to the core committers for approval.
 
 <div class="warning" markdown='1'>
-You should make the changes in the lowest branch they apply to. For instance, if you fix a spelling issue that you
-found in the 3.1 documentation, submit your fix to that branch in Github and it'll be copied to the master (3.2)
-version of the documentation automatically. *Don't submit multiple pull requests*.
+You should make your changes in the lowest branch they apply to. For instance, if you fix a spelling issue that you found in the 3.1 documentation, submit your fix to that branch in Github and it'll be copied to the master (3.2) version of the documentation automatically. *Don't submit multiple pull requests*.
 </div>
 
 ## Editing on your computer
 
-If you prefer to edit the content on your local machine, you can "[fork](http://help.github.com/forking/)" the 
+If you prefer to edit content on your local machine, you can "[fork](http://help.github.com/forking/)" the 
 [github.com/silverstripe/silverstripe-framework](http://github.com/silverstripe/silverstripe-framework) and 
-[github.com/silverstripe/silverstripe-cms](http://github.com/silverstripe/silverstripe-cms) repositories and send us 
-"[pull requests](http://help.github.com/pull-requests/)". If you have downloaded SilverStripe or a module, chances are 
-that you already have these repositories on your machine.
+[github.com/silverstripe/silverstripe-cms](http://github.com/silverstripe/silverstripe-cms) repositories, and send us "[pull requests](http://help.github.com/pull-requests/)" to incorporate your changes. If you have previously downloaded SilverStripe or a module, chances are that you already have these repositories on your machine.
```

Comment:
1`"pull request"` is linked here, but not earlier on in the article, I think this should be consistent